src/main/java/com/dougnoel/sentinel/webdrivers/WebDriverFactory.java

```diff
@@ -92,32 +94,57 @@ public static WebDriver instantiateWebDriver() {
         String downloadDirectory = ConfigurationManager.getOptionalProperty("download");
         if (downloadDirectory != null)
             DownloadManager.setDownloadDirectory(downloadDirectory);
+        
+        String runOnGrid = ConfigurationManager.getOptionalProperty("runOnGrid");
```

Comment:
After looking at the information needed, this variable is unnecessary. Just evaluate on the  `gridUrl ` value being set or not here and eliminate the use of the `turnGridOn` value.

Reply:
ok, make sense. will do that